Automattic/newspack-plugin
# Apply correct colour to the Jetpack logo

### All Submissions:






### Changes proposed in this Pull Request:

Jetpack has recently changed its green for a darker one. This PR updates it 😊

### How to test the changes in this Pull Request:

1. Go to the Setup Wizard, Integrations step
2. Check the current green
3. Switch to this branch
4. Refresh wizard -- you should see a darker green

### Other information:
